pkg/sql/pgwire/hba/hba.go

```diff
@@ -29,24 +34,56 @@ type Conf struct {
 }
 
 func (c Conf) String() string {
+	if len(c.Entries) == 0 {
+		return "# (empty configuration)\n"
+	}
 	var sb strings.Builder
+	table := tablewriter.NewWriter(&sb)
+	table.SetAutoWrapText(false)
+	table.SetReflowDuringAutoWrap(false)
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.SetBorder(false)
+	table.SetNoWhiteSpace(true)
+	table.SetTrimWhiteSpaceAtEOL(true)
+	table.SetTablePadding(" ")
+
+	row := []string{"# TYPE", "DATABASE", "USER", "ADDRESS", "METHOD", "OPTIONS"}
```

Comment:
While this happens to be the format of the `host` lines, the postgres hba.conf allows different connection types that can have other field types. For example if you want to add a socket type, the address column may not be needed. Can we avoid hard coding the column types for the entire table, since the spec allows for them to differ?